packages/web-components/src/player-component/player.class.ts

```diff
@@ -413,6 +417,8 @@ export class PlayerWrapper {
         this.player.addEventListener('error', this.onErrorEvent.bind(this));
         this.player.addEventListener('trackschanged', this.onTrackChange.bind(this));
         this.player.addEventListener('emsg', this.onShakaMetadata.bind(this));
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        (this.player as any).addEventListener('wallclock', this.onWallclock.bind(this));
```

Comment:
I am assuming this is temporary and this code will eventually move to [ava]player.ts in ShakaRTSP, otherwise we should add to the event map in shaka.ts (but not worth it if temporary because shaka.ts needs to be removed).

Reply:
Yes.  my hope is soon it will move to AVAPlayer and the widget doens't need the details about wallclock for RTSP vs vod